eq/client/systemWindow.h

```diff
@@ -77,6 +77,12 @@ class SystemWindow
     /** Bind the window's FBO, if it uses an FBO drawable. @version 1.0 */
     EQ_API virtual void bindFrameBuffer() const = 0;
 
+    /** Bind the window's draw FBO, used for multisampling. @version 1.9 */
+    EQ_API virtual void bindDrawFrameBuffer() const = 0;
+
+    /** Update the window's FBO from the multisampled FBO. @version 1.9 */
+    EQ_API virtual void updateFrameBuffer() const = 0;
```

Comment:
no. If at all, it's updateRenderFrameBuffer(), the FBO used by bindFrameBuffer().